start_jupyter_cm/kde.py

```diff
@@ -0,0 +1,137 @@
+import os
+import stat
+from shutil import copy
+
+homedir = os.environ['HOME']
+KPATH = os.path.join(homedir, ".local/share/kservices5/ServiceMenus")
+
+script = \
+    """#!/usr/bin/python
+
+import sys
+import os.path
+import subprocess
+
+folders = [path for path in sys.argv[1:] if os.path.isdir(path)]
+any_file_selected = len(folders) < len(sys.argv[1:])
+if any_file_selected:
+    subprocess.Popen(["jupyter-%s"])
+for folder in folders:
+    os.chdir(folder)
+    subprocess.Popen(["jupyter-%s"])
+    os.chdir("..")
+
+"""
+
+
+def get_script_file(terminal):
+    """
+    Returns the contents of a script file that will launch Jupyter
+    in a certain mode
+
+    Parameters
+    ----------
+    terminal: str
+        Either 'notebook' or 'qtconsole'
+    """
+    if terminal == 'qtconsole':
+        terminal += '-py2'
```

Comment:
On my system, this is necessary to get hyperspy in a python2 console. This should be changed, or another option should be added once we move to python3.